### PR TITLE
desktop: Add volume option to CLI

### PR DESCRIPTION
**Pull request by RasmusMathiesen**
Simple addition to let desktop users adjust global audio volume when starting ruffle from the command line.

### CODE REVIEW

File: desktop/src/main.rs [hunk 1]
```diff
@@ -109,6 +110,10 @@ struct Opt {
     #[clap(long, short, default_value = "show-all")]
     scale: StageScaleMode,
 
+    /// Audio volume
```
Comment (n0samu) [inline desktop/src/main.rs:113]: Let's add a little more information here - something like this:
`/// The audio volume as a number between 0 (muted) and 1 (full volume)`

**Review by n0samu — APPROVED** [review]
Welcome, and thank you for the contribution! Just have a little nit on the documentation.

**Comment by RasmusMathiesen** [comment]
Thank you :-) I added your suggestion.

File: desktop/src/main.rs [hunk 2]
```diff
@@ -109,6 +110,10 @@ struct Opt {
     #[clap(long, short, default_value = "show-all")]
     scale: StageScaleMode,
 
+    /// Audio volume as a number between 0 (muted) and 1 (full volume)
+    #[clap(long, short, default_value = "1.0")]
+    volume: f32,
```
Comment (relrelb) [inline desktop/src/main.rs:115]: Thinking out loud - Does it make more sense to define this as an integer in range 0-100? Feels a bit more user-friendly to me.

Comment (n0samu) [reply desktop/src/main.rs:115]: You would need to also make that change for the `volume` property of the player on web, and think about how to do that without causing unexpected behavior in people's existing sites... let's leave this PR as-is

**Review by relrelb — APPROVED** [review]
Thank you!

**Review by relrelb — APPROVED** [review]
Thank you!

**Pull request merged**